plugins/notifications-backend-module-email-notification-processor/README.md

```diff
@@ -0,0 +1,23 @@
+# @backstage/plugin-notifications-backend-module-email-notification-processor
+
+The email-notification-processor backend module for the notifications plugin.
+The idea is to send the notification details via email when the notification severity is above the configured threshold.
+
+Example configuration:
+
+```yaml
+backend:
+  email-notification-processor:
+    enabled: boolean # mandatory
```

Comment:
Don't think there is need for enabled/disabled flag. If user has installed this module, it should be enabled by default

Reply:
it is always good to have it. something can be temporarily broken and admin will need to disable it until problem resolved.

Reply:
It's the same as just commenting out the module from `packages/src/index.ts` and both of these require rebuild.

Reply:
editing code is not something you're supposed to do as admin. only config